src/main/java/fi/metatavu/soteapi/content/ContentController.java

```diff
@@ -51,8 +52,8 @@
    * @param category category slug
    * @return created content
    */
-  public Content createContent(String origin, String originId, String slug, ContentType contentType, Content parent, String category, Long orderIndex) {
-    return contentDAO.create(origin, originId, slug, contentType, parent, category, orderIndex, false);
+  public Content createContent(String origin, String originId, String slug, ContentType contentType, Content parent, String category, OffsetDateTime created, OffsetDateTime modified, Long orderIndex) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)